src/NovelRTUtilities.h

```diff
@@ -1,7 +1,19 @@
 // Copyright © Matt Jones and Contributors. Licensed under the MIT License (MIT). See LICENCE.md in the repository root for more information.
 
-#ifndef NOVELRT_NOVELRTMACROUTILITIES_H
-#define NOVELRT_NOVELRTMACROUTILITIES_H
+#ifndef NOVELRT_NOVELRTUTILITIES_H
+#define NOVELRT_NOVELRTUTILITIES_H
+
+namespace NovelRT {
+class NovelUtilities {
+public:
+  static const int SDL_SUCCESS = 0;
```

Comment:
These seems like things that would be better as enums or class constants...

Reply:
`SDL_SUCCESS` is universally used (@RubyNova said he'd want to switch to this for the other SDL calls that return a 0 on success) - but you'd mean to move the others to the class instead of just having them in a utilities class?

Reply:
Sorry, this was meant to apply to all the constants in here.

`SDL_SUCCESS` in particular seems like something that could be solved by a helper method. Namely something like `ThrowIfFailed(MyFunc(args), "Error Message")`

This would then check for success, log the error, and throw an exception.

Reply:
The others seem like they would be better exposed as class constants where they are relevant or as global constants exposed as part of the appropriate header file.

Reply:
Moved the constants to the header file for NovelAudioService (as they're related to the service or SDL_Mixer). Opened an issue for the helper method example with `SDL_SUCCESS`.